src/components/Settings/ProfilePopup.js

```diff
@@ -0,0 +1,214 @@
+import React, { useEffect, useState } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import {
+  Avatar,
+  Button,
+  CircularProgress,
+  Dialog,
+  DialogActions,
+  Slide,
+  Snackbar,
+  TextField,
+} from "@material-ui/core";
+import { getCurrentUserData } from "../../firebase/getCurrentUserData";
+import { updateCurrentUserData } from "../../firebase/updateCurrentUserData";
+import MuiAlert from "@material-ui/lab/Alert";
+import { storage } from "../../index";
+import Skeleton from "@material-ui/lab/Skeleton";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: "flex",
+    flexDirection: "column",
+    justifyContent: "center",
+    backgroundColor: "white",
+    maxWidth: 500,
+    height: "100%",
+    padding: theme.spacing(2),
```

Comment:
```suggestion
    padding: theme.spacing(3),
```
Dann ist das nicht so gedrängt und hat etwas mehr Freiraum. Kann man sich überlegen